pkg/executors/docker_compose_executor.go

```diff
@@ -177,6 +177,8 @@ func (e *DockerComposeExecutor) injectImagePullSecrets(callback EventHandler) in
 		switch s {
 		case api.ImagePullCredentialsStrategyDockerHub:
 			exitCode = e.injectImagePullSecretsForDockerHub(callback, c.EnvVars)
+    case api.ImagePullCredentialsStrategyECR:
+      exitCode = e.injectImagePullSecretsForECR(callback, c.EnvVars)
```

Comment:
Something is weird with the indentation here. If you are using vim, I recommend vim-go that automatically formats your source code.

Alternatively, you can call go format on the source tree.

Reply:
Look like my editor is converting <tabs> to <spaces> will run fmt to fix it.